ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -366,7 +366,7 @@ EOF
 
   cp "${dir}/bootstrap.ign" "${config_dir}/bootstrap_initial.ign"
   # We're using ancient fcct as glibc in installer container is 1.28
-  curl -sL https://github.com/coreos/butane/releases/download/v0.7.0/fcct-x86_64-unknown-linux-gnu >/tmp/fcct && chmod ug+x /tmp/fcct
+  curl -sL https://github.com/coreos/butane/releases/download/v0.7.0/fcct-"$(uname -m)"-unknown-linux-gnu >/tmp/fcct && chmod ug+x /tmp/fcct
```

Comment:
Lets add a function which does this and `$(uname -m | sed 's/aarch64/arm64/;s/x86_64/amd64/')` so that it would be reusable?

Reply:
+1, something like `gnu_arch() -> {x86_64, aarch64, ...}`, and `llvm_arch() -> {amd64, arm64, ...}` would make this clearer.

Reply:
I've added a new function where "sed" present and left the plain "uname -m" as is.

Reply:
To provide some context, 
- fcct uses x86_64 and aarch64. Here direct use of uname -m. It looks overkill to use a function here.
- yq uses amd64 and arm64. Here function is made use of.